pkg/cluster/nodeutils/util.go

```diff
@@ -99,7 +99,7 @@ func CopyNodeToNode(a, b nodes.Node, file string) error {
 
 // LoadImageArchive loads image onto the node, where image is a Reader over an image archive
 func LoadImageArchive(n nodes.Node, image io.Reader) error {
-	cmd := n.Command("ctr", "--namespace=k8s.io", "images", "import", "-").SetStdin(image)
+	cmd := n.Command("ctr", "--namespace=k8s.io", "images", "import", "--all-platforms", "-").SetStdin(image)
```

Comment:
do we know that image archives loaded at runtime by users will only contain the correct platform?
otherwise this is a performance regression

Reply:
I know the images we load during built are not like this, but this is not build time loading. this is runtime `kind load image-archive` etc.

Reply:
I got what you mean, I will remove this line.

Reply:
> I know the images we load during built are not like this, but this is not build time loading. this is runtime `kind load image-archive` etc.

I git what you mean.